autonomx/GameOfLife.cpp

```diff
@@ -143,42 +143,56 @@ void GameOfLife::computeIteration(double deltaTime)
 
     // compute iteration here
     if (iterationNumber % (100 - (int)(timeScale) + 1) == 0) {
-        for (int i = 1; i < latticeHeight - 1; i++) {
-            for (int j = 1; j < latticeWidth - 1; j++) {
+        for (int i = 0; i < latticeHeight; i++) {
+            for (int j = 0; j < latticeWidth; j++) {
 
                 // find index of the main cell, the cell just above it, and cell just below it; once we do that we can easily find the cells adjacent to those cells
-                //int index = i % latticeWidth + j * latticeWidth;
                 int index = i * latticeWidth + j;
                 int index_above = index-latticeWidth;
                 int index_below = index+latticeWidth;
-                //qDebug()<<"Top cell value" <<index;
-                // qDebug()<<"Middle cell value" <<index_above;
-                // int index_above = (i-1) % latticeWidth + j * latticeWidth;
-                // int index_below = (i+1) % latticeWidth + j * latticeWidth;
 
                 // finding the total value of all the neighbors near to the main cell
-                int neighbors = cells[index_above-1] + cells[index_above] + cells[index_above+1]
-                        + cells[index-1] + cells[index+1]
-                        + cells[index_below-1] + cells[index_below] + cells[index_below+1];
-
-                /* int neighbors = cells[index - latticeWidth - 1] + cells[index - latticeWidth] + cells[index - latticeWidth + 1]
-                        + cells[index - 1] + cells[index] + cells[index + 1]
-                        + cells[index + latticeWidth - 1] + cells[index + latticeWidth] + cells[index + latticeWidth + 1]; */
-
-                /*   // declare the cell alive or dead based on the rule below
-                if      ((cells[index] == 1) && (neighbors <  2)) cells[index] = 0;
-                //TO DO: Here neighbors should be 3 infront of 4 in the just below line. However, changed to 4 to make the GOL look decent. Artistic Lisence. Can discuss.
-                else if ((cells[index] == 1) && (neighbors >  3)) cells[index] = 0;
-                else if ((cells[index] == 0) && (neighbors == 3)) cells[index] = 1; */
-
-                // declare the cell alive or dead based on the rule below
+                neighbors = 0; //initializing a default value
+                // if the cells are on the extreme left of the lattice consider the cells on the extreme right as their neighbors :ignoring the 4 corner cells
+                if (j==0 && i!=0 && i!=latticeHeight-1){
```

Comment:
The rest of the code in the whole program has more space between each operator. It would be best if the coding style for this part would be similar.

For example: `2 + 2`, not `2+2`